blackbox/docs/administration/hba.txt

```diff
@@ -0,0 +1,108 @@
+.. _administration_hba:
+
+===============================
+Host Based Authentication (HBA)
+===============================
+
+This section explains how to configure CrateDB client connection and authentication.
+
+By default, the optional setting ``auth.host_based`` is absent and therefore host based
+authentication is disabled. In this instance, the CrateDB cluster allows any
+unauthenticated connections.
+
+To allow authenticated access to CrateDB from specific hosts, you need to configure
+the ``auth.host_based`` setting in the ``crate.yml`` file.
+
+.. NOTE::
+
+   For the authentication feature to work properly, ``auth.host_based`` setting
+   configuration needs to be the same for every eligible master node. However,
+   it is possible to update the cluster state with the new configuration by
+   restarting only a single node of the cluster, because the last node that
+   joins the cluster overrides the existing host based authentication
+   configuration in the cluster state.
+
+Allowing Authentication to CrateDB
+==================================
+
+.. NOTE::
+
+   *Host Based Authentication* (HBA) is an Enterprise Edition feature.
```

Comment:
pls keep the Enterprise note on top, just the postgresql restriction here.
the point is that the whole thing is enterprise, but we "currently" only have pg proto support